### PR TITLE
frontend: update address type switch

### PR DESCRIPTION
**Pull request by thisconnect**
In the next design revamp the address type UI will probably be a
checkbox, keep changing address type as toggle until the revamp.

small drawback, the labels will probably not be "change to …" but rather just what it is "modern address..."

### CODE REVIEW

**Comment by thisconnect** [comment]
I didn't test with the latest firmware, only patched receive-addresses to fake p2tr addresses and simulate taproot support.

**Comment by thisconnect** [comment]
It didn't lint because the key's were not in alphabetical order, fixed+squashed, ci should pass now.

@benma PTAL

should go into https://github.com/digitalbitbox/bitbox-wallet-app/pull/1639

File: frontends/web/src/routes/account/receive/receive.tsx [hunk 1]
```diff
@@ -246,13 +250,10 @@ class Receive extends Component<Props, State> {
...
                 { this.availableScriptTypes.length > 1 && (
-                      this.availableScriptTypes.map(scriptType => (
-                          <p className={style.changeType} onClick={() => this.switchToScriptType(scriptType)}>
-                              {t(`receive.scriptType.${scriptType}`)}
-                              { scriptType === receiveAddresses[addressType].scriptType && ' (active)' }
-                          </p>
-                      ))
-                  )}
+                    <button className={style.changeType} onClick={this.nextAddressType}>
+                        {t(`receive.scriptType.${receiveAddresses[this.getNextAddressTypeIndex(addressType)].scriptType}`)}
```
Comment (benma) [inline frontends/web/src/routes/account/receive/receive.tsx:254]: or just `${this.availableScriptTypes[this.getNextAddressTypeIndex(addressType)]}`? less indirection.

Comment (benma) [reply frontends/web/src/routes/account/receive/receive.tsx:254]: I added your commit and applied this here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1639

**Comment by benma** [comment]
Thanks - applied!

**Pull request closed without merging**